embrace_mobile/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": "2.0.0",
   "app_uuid": "86988058-9b89-45a8-b92f-5473a96e4a36",
   "app_id": "embrace-mobile",
-  "display_on_public_website": true,
+  "display_on_public_website": false,
```

Comment:
We should keep this as true since the tile will still be live.

Reply:
Good catch,I was under the impression this was only a UI Extension but looks like it emits metrics. 
I'll cut a PR to update the README similar to what we did for the others.

Reply:
You had it correct in the gdoc

Reply:
Thanks for helping clarify things (here and offline) I had conflicting notes (that I wrote) in that doc so I got confused!